setup.py

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of Viper - https://github.com/viper-framework/viper
+# See the file 'LICENSE' for copying permission.
+from __future__ import unicode_literals
```

Comment:
I would feel a lot better about with the whole String/Unicode handling (while maintaining support for Python 2.7)  if we had tests that prove that stuff that was working are still doing the same thing on Python 3. See also my remark on absolute_import.